README.md

```diff
@@ -142,6 +142,27 @@ document = client.create_document(document: document_create)
 documents = client.create_documents(documents: [document_create1, document_create2])
 ```
 
+## Command Line Interface
+
+This gem includes a `send_to_readwise` command-line tool for quickly sending HTML content to Readwise Reader.
```

Comment:
(or it could be `readwise`)

Reply:
I'm thinking `readwise document create`, similar to how the `gh` CLI is structured: https://cli.github.com/manual/gh_project_create